assets/styles/components/molecules/SfPagination.scss

```diff
@@ -0,0 +1,58 @@
+.sf-pagination {
+  display: flex;
+  align-items: center;
+
+  &__item {
+    --link-text-decoration: none;
+    --pagination-item-width: 1.5rem;
+    --pagination-item-height: 1.375rem;
```

Comment:
all variables should be defined in assets/style/variables only. Instead of creating new variables use spacers only.

Reply:
Resolved